elife/composer.sls

```diff
@@ -80,15 +78,14 @@ composer-global-paths:
         - require:
             - file: composer-home-dir
 
-# lsh@2021-05-25: requisites would seem to preclude this from running:
-# 'run before composer is installed' and 'only if composer is installed'
-# todo: fix or remove
 update-composer:
```

Comment:
might even be able to get rid of this state altogether - specifying `--version` in install-composer seems to be enough